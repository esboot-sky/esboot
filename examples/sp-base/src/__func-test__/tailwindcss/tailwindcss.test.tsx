import { render } from '@testing-library/react';
import { test, expect } from 'vitest';

import Tailwindcss from './tailwindcss';

test('Tailwindcss is working', () => {
  const { container, getByText, getByTestId } = render(<Tailwindcss />);
  expect(getByText('Tailwindcss')).toBeInTheDocument();

  expect(container).toMatchSnapshot();

  const element = getByTestId('tailwindcss');
  expect(element).toHaveStyle('color: white');
});
