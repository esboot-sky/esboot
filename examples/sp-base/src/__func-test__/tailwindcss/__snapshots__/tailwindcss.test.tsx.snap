// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Tailwindcss is working 1`] = `
<div>
  <div
    class="bg-red-500 text-white"
    data-testid="tailwindcss"
  >
    Tailwindcss
  </div>
</div>
`;
